source/2020-07-29-ember-3-20-released.md

```diff
@@ -0,0 +1,133 @@
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.20
+
+Ember.js 3.20 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### New Features (1)
+
+##### `{{in-element}}`
+
+Have you ever wished you could render a component somewhere outside of the usual HTML flow? To help with this, Ember.js now provides the `{{in-element}}` helper as public API. If you have used community addons like [ember-wormhole](https://github.com/yapplabs/ember-wormhole) or [ember-elsewhere](https://github.com/ef4/ember-elsewhere), you are probably familiar with the challenge that `{{in-element}}` aims to solve.
+
+To use the helper, pass in a DOM element to target (`this.myDestinationElement` in the example below) and a block to render:
+
+```handlebars
+{{#in-element this.myDestinationElement}}
+  <div>Some content</div>
+{{/in-element}}
+```
+
+This new public API varies somewhat compared to the behavior of the private API:
+
+- For the public API `{{in-element}}`, by default, the rendered content will replace all the content of the destination, effectively becoming the its `innerHTML`. If you want it to be appended instead of replacing the content, you can pass in `insertBefore=null`.
+- In the private API `{{-in-element}}`, the rendered content was appended to any existing content in the destination.
+
+Developers should use the public API, `{{in-element}}`, and discontinue using `{{-in-element}}`.
+
+See [RFC 287](https://emberjs.github.io/rfcs/0287-promote-in-element-to-public-api.html) for more information about the motivation for this new feature.
+
+#### Deprecations (0)
+
+No new deprecations are introduced in this release.
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.20, please review the [Ember.js 3.20.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.20.0).
```

Comment:
There _was_ a non-trivial deprecation for `Meta.prototype.setDestroying` that was added in 3.20. To avoid this deprecation you will need the latest versions of `@glimmer/component` and `ember-modifier`.

See https://github.com/emberjs/ember.js/issues/19042 for links/details/etc.

Reply:
Ok thanks, adding that now!